shoppingm8_fe/lib/main.dart

```diff
@@ -31,7 +31,7 @@ class MyApp extends StatelessWidget {
         // Notice that the counter didn't reset back to zero; the application
         // is not restarted.
         primarySwatch: Colors.green,
-        backgroundColor: Color(0xFFDDFFEE),
+        backgroundColor: Color.fromARGB(255, 225, 255, 225),
```

Comment:
Is the color change intentional? If so, couldn't it be replaced with Color.white?